### ISSUE TITLE
Excluded directories: wildcards

### ISSUE THREAD

**Issue by RedFred7** [DoTheEvo/ANGRYsearch#67]
I love AngrySearch, but it dosn't seem to parse wildcards in the excluded directories setting. For instance, I set "$HOME/.*" but it seems to be totally ignored. Is this a bug or just a not-implemented yet feature ?

**Comment by nighthawk9** [comment]
I don't believe this a feature. Looking through the code, any path with the given value is removed. 

For example if a path /var/ is excluded, if this is seen it excludes all full paths that include /var/  

I could be mistaken. I was going to implement this, if the author is still active.
